src/UnderlineNav2/UnderlineNav.tsx

```diff
@@ -325,7 +325,10 @@ export const UnderlineNav = forwardRef(
                   onClick={onAnchorClick}
                   trailingIcon={TriangleDownIcon}
                 >
-                  More
+                  <Box as="span" aria-hidden="true">
+                    More
+                  </Box>
+                  <VisuallyHidden as="span">{`More: ${ariaLabel} items`}</VisuallyHidden>
```

Comment:
I think it could be helpful to structure this like:

```suggestion
                  <Box as="span">
                    More <VisuallyHidden as="span">{`${ariaLabel} items`}</VisuallyHidden>
                  </Box>
                  
```

This could help to avoid issues identified in: https://www.w3.org/WAI/WCAG22/Techniques/general/G211 where the accessible and visual label differ and make sure that they at least include a common base (in this case, text: "More")

Curious what you think @broccolinisoup @ericwbailey 👀

Reply:
😮 I LOVE IT SO MUCH

Reply:
SAME